main.go

```diff
@@ -54,7 +54,7 @@ func getEnvInt64(key string, defaultVal int64) int64 {
 
 func main() {
 	var (
-		redisAddr            = flag.String("redis.addr", getEnv("REDIS_ADDR", "redis://localhost:6379"), "Address of the Redis instance to scrape")
```

Comment:
Please leave this as is, this might break things for people as they didn't have to set this in the path to pick up a local redis instance.